src/styles/main.scss

```diff
@@ -1 +1,58 @@
 // put code here
+
+@keyframes rotate360 {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.title {
+  text-align: center;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+
+.clock {
+  height: 80vmin;
+  width: 80vmin;
+  border-radius: 80vmin;
+  border: 10px dotted rgb(160, 0, 0);
+  background: rgb(0, 0, 0);
+  position: relative;
+  margin: 0 auto;
+
+  &__hours {
+    height: 20vmin;
+    width: 2vmin;
+    background: red;
+    position: absolute;
+    top: 20vmin;
+    left: 39vmin;
+    transform-origin: 1vmin 20vmin;
+    animation: 60s linear infinite rotate360;
+  }
+
+  &__minutes {
+    height: 40vmin;
+    width: 1vmin;
+    background: red;
+    position: absolute;
+    top: 1vmin;
+    left: 39.5vmin;
+    transform-origin: 0.5vmin 39vmin;
+    animation: 60s linear infinite rotate360;
```

Comment:
this arrow should make a circle in 15 seconds, not 60

Reply:
https://github.com/mate-academy/layout_stop-watch/pull/2822#discussion_r1350645152 - your colleague wrote something different, can you explain > ?